modules/ROOT/partials/configuration/advanced-typography.adoc

```diff
@@ -0,0 +1,250 @@
+
+[source,js,subs="attributes+"]
+----
+tinymce.init({
+  selector: "textarea",
+  plugins: "typography",
+  toolbar: "typography",
+  typography_langs: [ "en-US", "de", "es" ], // Required to set specific typography language rules.
+});
+</script>
+----
+
+// Test Fiddle used by QA team for reference: https://fiddle.tiny.cloud/uqiaab/58
+
+[[typography_custom_rules]]
+== `+typography_custom_rules+`
+
+// What this option does, precisely.
+// questions still about `typography_custom_rules`.
+// from the supplied example it is unclear (to me, at least) what it does.
+// is `/"/g, "'"` just `sed` and `regex`?
+// and, even if so, I remain unclear as to what
+//     'en/quotes': (text) => text.replace(/"/g, "'"),
+// does to
+//     (text)
+// other than replace ? it with
+//     text.replace(/"/g, "'")
+
+*Type:* `+Object+`
+
+=== Example: Using `typography_custom_rules`
+
+[source,js,subs="attributes+"]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: '{plugincode}',
+  toolbar: '{plugincode}',
+  language: 'en_US', // This is required to configure the default language used by typography.
+  typography_custom_rules: {
+    'en/quotes': (text) => text.replace(/"/g, "'"),
+  },
+});
+----
+
+
+[[typography_ignore]]
+== `+typography_ignore+`
+
+This option tells the {pluginname} plugin to ignore content within specified HTML tags or within specified HTML tags that have specified attributes.
+
+If an HTML tag or an HTML tag with a specified attribute is added to the `typography_ignore` array, the Typograhy plugin’s rules are not applied to material set within that tag.
+
+*Type:* `+Array+`
+
+*Possible values:* `[ '<html-selector>', 'span["<html-selector>"]' ]`
+
+*Default value:* none
+
+=== Example: Using `typography_ignore`
+
+[source,js,subs="attributes+"]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: '{plugincode}',
+  toolbar: 'typography',
+  language: 'en_US', // This is required to configure the default language used by typography.
+  typography_ignore: [ 'code', 'span["lang"]' ]
+});
+----
+
+
+[[typography_rules]]
+== `+typography_rules+`
+
+This option specifies which rules are available to the {pluginname} plugin.
+
+The {pluginname} plugin rules are sourced from the https://github.com/typograf/typograf[typograf] library.
+
+All rules supported by the {pluginname} plugin are documented in the https://github.com/typograf/typograf/blob/dev/docs/RULES.en-US.md[Rules of typograf].
+
+*Type:* `+Array+`
+
+=== Example: Using `typography_rules`
+
+[source,js,subs="attributes+"]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: '{plugincode}',
+  toolbar: '{plugincode}',
+  language: 'en_US', // This is required to configure the default language used by typography.
+  typography_rules: {
+    en: ['en/quotes', 'en/spaces']
+    es: ['es/quotes', 'es/spaces']
+    }
```

Comment:
Should be an array of rules, we do not support language-specific rules, yet.
```js
  typography_rules: [
    'common/punctuation/quote',
    'en-US/dash/main',
    'common/nbsp/afterParagraphMark',
    'common/nbsp/afterSectionMark',
  ]
```

Reply:
To quote the commit message:

> Changed section of `typography_rules` example code to array, as per @yacodes note.